obs_img_utils/utils.py

```diff
@@ -146,60 +146,59 @@ def echo_style(message, no_color, fg='green'):
         click.secho(message, fg=fg)
 
 
-def conditions_repl():
+def conditions_repl(no_color):
     """
-    Query and accept input for image condition options.
+    Query and accept input for condition options.
     """
     image_conditions = []
     while True:
-        if click.confirm('Add an image condition?'):
-            condition_type = click.prompt(
-                'Enter the condition type',
-                type=click.Choice(['image', 'package'])
+        if click.confirm('Add a condition?'):
+            package_name = click.prompt(
+                'Enter the package name '
+                '(leave blank for an image condition)',
+                type=str,
+                default=''
             )
 
-            if condition_type == 'image':
-                image_version = click.prompt(
-                    'Enter the image version condition',
-                    type=str
-                )
-
-                image_conditions.append({'image': image_version})
-            else:
-                package_name = click.prompt(
-                    'Enter the package name',
-                    type=str
-                )
+            condition_exp = click.prompt(
+                'Enter the condition expression',
+                type=click.Choice(['>=', '<=', '==', '>', '<']),
+                default='>='
+            )
 
-                condition_exp = click.prompt(
-                    'Enter the condition expression',
-                    type=click.Choice(['>=', '<=', '==', '>', '<']),
-                    default='>='
-                )
+            version = click.prompt(
+                'Enter the version (optional)',
+                type=str,
+                default=''
+            )
 
-                version = click.prompt(
-                    'Enter the version (optional)',
-                    type=str,
-                    default=''
-                )
+            release = click.prompt(
+                'Enter the release (optional)',
+                type=str,
+                default=''
+            )
 
-                release = click.prompt(
-                    'Enter the release (optional)',
-                    type=str,
-                    default=''
+            if not (package_name or version or release):
+                echo_style(
+                    'Condition skipped, conditions require at least one of:'
+                    ' package_name, version or release.',
```

Comment:
Shouldn't it require `package_name` _and_ either `version` or `release` ?

Reply:
If package_name is left empty the condition is for the image itself:

```
Enter the package name (leave blank for an image condition) []:
```

Reply:
Full usage example to set a kiwi version condition on the image:

```
Add a condition? [y/N]: y
Enter the package name (leave blank for an image condition) []: 
Enter the condition expression (>=, <=, ==, >, <) [>=]: 
Enter the version (optional) []: 8.13.21
Enter the release (optional) []: 
Add a condition? [y/N]: n
```